Makefile

```diff
@@ -114,6 +114,9 @@ test_rpmbuild: buildrpm_image
 test_rpmbuild_cockpit-composer: buildrpm_image
 	sudo docker run --rm --name buildrpm -v `pwd`:/welder welder/buildrpm:latest make cockpit-composer-rpm cockpit-composer-srpm
 
+debug_install:
+	sudo rm -rf /usr/share/cockpit/welder && sudo cp -r `pwd`/public /usr/share/cockpit/welder
```

Comment:
IMHO this is wrong, as it scribbles over package manager territory. It's much better to create a ~/.local/share/cockpit/welder symlink to `$(pwd)/public`. Maybe call it "local-install"?

Also, this new target needs to be added to `.PHONY`.